### PR TITLE
Source: Facebook Marketing use latest version of facebook_bussiness dep lib

### LINKED ISSUE #3991: Setup Facebook Marketing source fails
## Expected Behavior
After configurated the setting of Facebook Marketing source, it should be able to set up the source sucessfully.

## Current Behavior
It is now fails during the connection tests. It returns
`Error: 2635, (#2635) You are calling a deprecated version of the Ads API. Please update to the latest version: v11.0.`

## Logs
```
2021-06-09 13:20:59 INFO (/tmp/workspace/09cb9090-1f85-4ae5-892a-5649dfbbe57c/0) TemporalAttemptExecution(get):111 - Executing worker wrapper. Airbyte version: 0.24.7-alpha
2021-06-09 13:20:59 INFO (/tmp/workspace/09cb9090-1f85-4ae5-892a-5649dfbbe57c/0) LineGobbler(voidCall):85 - Checking if airbyte/source-facebook-marketing:0.2.7 exists...
2021-06-09 13:20:59 INFO (/tmp/workspace/09cb9090-1f85-4ae5-892a-5649dfbbe57c/0) LineGobbler(voidCall):85 - airbyte/source-facebook-marketing:0.2.7 was found locally.
2021-06-09 13:20:59 INFO (/tmp/workspace/09cb9090-1f85-4ae5-892a-5649dfbbe57c/0) DockerProcessFactory(create):111 - Preparing command: docker run --rm --init -i -v airbyte_workspace:/data -v /tmp/airbyte_local:/local -w /data/09cb9090-1f85-4ae5-892a-5649dfbbe57c/0 --network host airbyte/source-facebook-marketing:0.2.7 check --config source_config.json
2021-06-09 13:21:00 ERROR (/tmp/workspace/09cb9090-1f85-4ae5-892a-5649dfbbe57c/0) DefaultAirbyteStreamFactory(internalLog):108 - Error: 2635, (#2635) You are calling a deprecated version of the Ads API. Please update to the latest version: v11.0.
2021-06-09 13:21:00 ERROR (/tmp/workspace/09cb9090-1f85-4ae5-892a-5649dfbbe57c/0) DefaultAirbyteStreamFactory(internalLog):108 - Check failed
2021-06-09 13:21:00 INFO (/tmp/workspace/09cb9090-1f85-4ae5-892a-5649dfbbe57c/0) TemporalAttemptExecution(get):134 - Stopping cancellation check scheduling...
```

## Steps to Reproduce
1. Click set up the source with source type facebook marketing
2. Fill in all the configurations then click set up source

## Severity of the bug for you
High

## Airbyte Version
0.24.7-alpha

## Connector Version (if applicable)
ource-facebook-marketing:0.2.7

## Additional context
It is a newly created environment that I wanted to try Airbyte features.

### PR DESCRIPTION
**Pull request by marcosmarxm**
## What
Closes #3991 

## How
*Describe the solution*

## Recommended reading order
1. `x.java`
2. `y.python`

## Pre-merge Checklist
Expand the checklist which is relevant for this PR. 

<details><summary> <strong> Connector checklist </strong> </summary>
<p>

- [ ] Issue acceptance criteria met
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.io/contributing-to-airbyte/updating-documentation#issues-and-pull-requests)
- [ ] Secrets are annotated with `airbyte_secret` in output spec
- [ ] Unit & integration tests added as appropriate (and are passing)
    * Community members: please provide proof of this succeeding locally e.g: screenshot or copy-paste acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] `/test connector=connectors/<name>` command as documented [here](https://docs.airbyte.io/contributing-to-airbyte/building-new-connector#updating-an-existing-connector) is passing. 
    * Community members can skip this, Airbyters will run this for you. 
- [ ] Code reviews completed
- [ ] Credentials added to Github CI if needed and not already present. [instructions for injecting secrets into CI](https://docs.airbyte.io/contributing-to-airbyte/building-new-connector#using-credentials-in-ci). 
- [ ] Documentation updated 
    - [ ] README
    - [ ] CHANGELOG.md
    - [ ] Reference docs in the `docs/integrations/` directory.
    - [ ] Build status added to [build page](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/builds.md)
- [ ] Build is successful
- [ ] Connector version bumped like described [here](https://docs.airbyte.io/contributing-to-airbyte/building-new-connector#updating-a-connector)
- [ ] New Connector version released on Dockerhub by running the `/publish` command described [here](https://docs.airbyte.io/contributing-to-airbyte/building-new-connector#updating-a-connector)
- [ ] No major blockers
- [ ] PR merged into master branch
- [ ] Follow up tickets have been created
- [ ] Associated tickets have been closed & stakeholders notified
</p>
</details>

<details><summary> <strong> Connector Generator checklist </strong> </summary>
<p>
   
- [ ] Issue acceptance criteria met
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.io/contributing-to-airbyte/updating-documentation#issues-and-pull-requests)
- [ ] If adding a new generator, add it to the [list of scaffold modules being tested](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connector-templates/generator/build.gradle#L41)
- [ ] The generator test modules (all connectors with `-scaffold` in their name) have been updated with the latest scaffold by running `./gradlew :airbyte-integrations:connector-templates:generator:testScaffoldTemplates` then checking in your changes
- [ ] Documentation which references the generator is updated as needed.
</p>
</details>

### CODE REVIEW

**Comment by marcosmarxm** [comment]
@keu I updated `facebook_bussiness` to version **11.0**. Some users are having problem setting new connection (see image).
Tests failed because of rate limit or maybe there other change related to this version upgrade. Questioning you because I saw you open a PR in the fb library
image

File: airbyte-config/init/src/main/resources/config/STANDARD_SOURCE_DEFINITION/e7778cfc-e97c-4458-9ecb-b4f2bba8946c.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "sourceDefinitionId": "e7778cfc-e97c-4458-9ecb-b4f2bba8946c",
   "name": "Facebook Marketing",
   "dockerRepository": "airbyte/source-facebook-marketing",
-  "dockerImageTag": "0.2.9",
+  "dockerImageTag": "0.2.10",
```
Comment (keu) [inline airbyte-config/init/src/main/resources/config/STANDARD_SOURCE_DEFINITION/e7778cfc-e97c-4458-9ecb-b4f2bba8946c.json:5]: @marcosmarxm I think you forgot to update Dockerfile

**Review by keu — APPROVED** [review]
small fix, LGTM otherwise

File: airbyte-config/init/src/main/resources/config/STANDARD_SOURCE_DEFINITION/e7778cfc-e97c-4458-9ecb-b4f2bba8946c.json (hunk 1 above)
Comment (marcosmarxm) [reply airbyte-config/init/src/main/resources/config/STANDARD_SOURCE_DEFINITION/e7778cfc-e97c-4458-9ecb-b4f2bba8946c.json:5]: thanks!

**Comment by keu** [comment]
It is weird that we don't have tests failing because of deprecated ads lib.

**Comment by marcosmarxm** [comment]
Just trying to run tests again, after that I'm going to release the new version.

File: airbyte-integrations/connectors/source-facebook-marketing/unit_tests/test_client.py [hunk 2]
```diff
@@ -50,7 +50,7 @@ def mock_default_sleep_interval(mocker):
 @pytest.fixture(name="client")
 def client_fixture(some_config, requests_mock, fb_account_response):
     client = Client(**some_config)
-    requests_mock.register_uri("GET", FacebookSession.GRAPH + "/v10.0/me/adaccounts", [fb_account_response])
+    requests_mock.register_uri("GET", FacebookSession.GRAPH + "/v11.0/me/adaccounts", [fb_account_response])
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-facebook-marketing/unit_tests/test_client.py:53]: I think we should use `FacebookAdsApi.API_VERSION` this dependency is not good :/

**Comment by davinchia** [comment]
looks like FB deprecated some fields: https://developers.facebook.com/docs/graph-api/changelog/version11.0#insights-api

File: airbyte-integrations/connectors/source-facebook-marketing/Dockerfile [hunk 3]
```diff
@@ -12,5 +12,5 @@ RUN pip install .
 ENV AIRBYTE_ENTRYPOINT "python /airbyte/integration_code/main.py"
 ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
 
-LABEL io.airbyte.version=0.2.9
+LABEL io.airbyte.version=0.2.9=10
```
Comment (tuliren) [inline airbyte-integrations/connectors/source-facebook-marketing/Dockerfile:15]: This is a typo, right?

```suggestion
LABEL io.airbyte.version=0.2.10
```

**Review by tuliren — APPROVED** [review]

**Comment by jaimefr** [comment]
It is not working properly because the new API version deprecated some more fields than the ones removed by @davinchia . I'm getting this error:
```
Status: 400
Response:
{
"error": {
"message": "(#100) store_visit_actions, cost_per_store_visit_action are not valid for fields param. please check https://developers.facebook.com/docs/marketing-api/reference/ads-insights/ for all valid values",
"type": "OAuthException",
"code": 100,
```

They need to be removed from airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/schemas/ads_insights.json and all airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/schemas/ads_insights_*.json

**Pull request merged**